Content.Shared/Cargo/Components/PriceGunComponent.cs

```diff
@@ -0,0 +1,9 @@
+namespace Content.Shared.Cargo.Components;
+
+/// <summary>
+///     This is used for the price gun, which calculates the price of any object it appraises.
+/// </summary>
+[RegisterComponent]
```

Comment:
networkedcomp

Reply:
Its either forgetting networking or spelling mistakes with me... I don't think I've had a PR without one of those 😆